### DISCUSSION TITLE
Aggregated charge at a location by period

### DISCUSSION THREAD

**Discussion by jyothri** [teslamate-org/teslamate#3314] [Ideas]
Hi,

My primary use case: I would like to see how much I charge in a month at home. This can help extrapolate for e.g. how much solar panel size I need for my house. 
As a more generic FR it would be helpful to view the KWh charged per location (home or work) in a period (week, month, year).

NOTE: 
I looked at the `statistics` dashboard which shows overview at month but cannot drill it down by location.
I can filter by Geofence location in the `charges` dashboard but have to manually aggregate for the time period.

**Comment by cwanja** [comment]
Have you looked at Charging stats dashboard with defined timeline and the Geofence charging location? You still have to define the timeline like the Charges dashboard. 

You could also create your own dashboard. I would think starting with Statistic and adding a Geofence filter would be the easiest method.
